src/block/Block.php

```diff
@@ -642,4 +643,8 @@ public function calculateIntercept(Vector3 $pos1, Vector3 $pos2) : ?RayTraceResu
 
 		return $currentHit;
 	}
+
+	public function getSupportType(int $facing) : SupportType{
+		return SupportType::NONE();
```

Comment:
I'm not sure this really makes sense. `Block` assumes the presence of a 1x1x1 collision cube, which normally equates to full support.

Reply:
Indeed, this part should be as ``Full``, and the ``Transparent`` should be modified for a none support